remote.go

```diff
@@ -466,6 +466,10 @@ func (wd *remoteWebDriver) SwitchFrame(frame string) error {
 	return wd.voidCommand("/session/%s/frame", params)
 }
 
+func (wd *remoteWebDriver) SwitchFrameParent() error {
+	return wd.voidCommand("/session/%s/frame/parent", nil)
```

Comment:
I am on a plane and have forgotten much of the context of this code, so excuse me if this is obvious—but is this the correct URL? Do you need to be able to pass in a frame index? Won't the `%s` expand to nil?